public/js/dashboard.js

```diff
@@ -773,6 +773,16 @@ app.controller('badmintonCGController', ['$scope', 'socket',
             $scope.badminton = msg;
         });
 
+        $scope.reset1 = function() {
```

Comment:
If reset could only reset the current set of points (match score) and not the overall game score that would be better.